src/werkzeug/routing.py

```diff
@@ -1741,6 +1778,10 @@ def match(self, path_info=None, method=None, return_rule=False, query_args=None)
           You can use the `RequestRedirect` instance as response-like object
           similar to all other subclasses of `HTTPException`.
 
+        - you recieve a `WebsocketMissmatch` exception if the only match is
```

Comment:
shouldn't those names here use double-backticks (also above for `RequestRedirect`) since it's rst?

Reply:
I changed this, but the text in general uses both.